tests/php/Http/Controllers/Api/VolumeControllerTest.php

```diff
@@ -156,23 +161,27 @@ public function testUpdateUrl()
         $this
           ->json('PUT', '/api/v1/volumes/'.$this->volume()->id, [
             'url' => 'admin-test://volumes',
-            ])->assertStatus(422);
+            ])
+            ->assertStatus(422);
```

Comment:
Somehow the indentation is off here (and in the lines below). I'm sure CS Fixer will complain about this, too, so maybe just run it again.